### PR TITLE
🔧Housekeeping for setup

### PR DESCRIPTION
**Pull request by brandon-hedrick**
* Add prisma as a dependency. 
* Update .gitignore
* Add node version to `engines` object in `package.json`

### CODE REVIEW

**Comment by brandon-hedrick** [comment]
Squash and merge when we're done

File: README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@
 
 ### First time setup
 
+_Note_ this project requires `Node 8.1` or later.
```
Comment (markbrouch) [inline README.md]: Instead of documenting this, let's add the required node engine to package.json

Comment (markbrouch) [reply README.md]: https://docs.npmjs.com/files/package.json#engines

File: package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
     "isomorphic-unfetch": "^2.0.0",
     "jsonwebtoken": "^8.2.2",
     "next": "^6.0.3",
+    "prisma": "^1.9.0",
```
Comment (markbrouch) [inline package.json:31]: good call, I had it globally installed as per the prisma docs, but this is better

**Review by markbrouch — APPROVED** [review]

**Pull request merged**